reader/package.json

```diff
@@ -4,19 +4,27 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "jest"
+    "test": "jest",
+    "migrate": "babel-node -r dotenv/config ./lib/migrations/index.js",
+    "new-migration": "./bin/new-migration.js"
   },
-  "type": "module",
   "author": "ralph",
   "license": "WTFPL",
   "devDependencies": {
-    "@babel/preset-env": "^7.16.5",
+    "@babel/core": "^7.19.3",
+    "@babel/preset-env": "^7.19.4",
+    "babel-jest": "^29.1.2",
     "commander": "^9.3.0",
     "dotenv": "^10.0.0",
     "eslint": "^8.14.0",
     "googleapis": "^92.0.0",
     "jest": "^27.4.5",
     "node-fetch": "^3.1.0",
     "slugify": "^1.6.4"
+  },
+  "dependencies": {
```

Comment:
do any of these need to be added as dependencies? Fine if they do

Reply:
probably not, I mean this isn't deployed anywhere right?